src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -769,6 +771,252 @@ protected bool IsAlive
+        /// </summary>
+        /// <returns>Child Elements.</returns>
+        public IEnumerable<Element> GetChildElements()
+        {
+            return GetElementChildElements(this.InternalElement);
+            
+        }
+
+        private IEnumerable<Element> GetElementChildElements(Autodesk.Revit.DB.Element element)
+        {
+            List<Element> components = new List<Element>();
+            BuiltInCategory builtInCategory = (BuiltInCategory)System.Enum.Parse(typeof(BuiltInCategory),
+                                                                                 element.Category.Id.ToString());
+
+            switch (builtInCategory)
+            {
+                case BuiltInCategory.OST_Stairs:
+                    List<Element> stairComponentElements = GetChildElementsFromStairs(element);
+                    components.AddRange(stairComponentElements);
+                    break;
+
+                case BuiltInCategory.OST_StructuralFramingSystem:
+                    List<Element> beamSystemComponentElements = GetChildElementsFromStructuralFramingSystem(element);
+                    components.AddRange(beamSystemComponentElements);
+                    break;
+
+                case BuiltInCategory.OST_StairsRailing:
+                    List<Element> railingComponentElements = GetChildElementsFromRailings(element);
+                    components.AddRange(railingComponentElements);
+                    break;
+
+                default:
+                    List<Element> componentElements = GetChildElementsFromFamilyInstance(element);
+                    components.AddRange(componentElements);
+                    break;
+            }
+            return components;
+        }
+
+        private static List<Element> GetChildElementsFromFamilyInstance(Autodesk.Revit.DB.Element element)
+        {
+            var familyInstance = element as Autodesk.Revit.DB.FamilyInstance;
+            if (familyInstance == null)
+                throw new NullReferenceException(Properties.Resources.NoChildElements);
+            List<ElementId> componentIds = familyInstance.GetSubComponentIds().ToList();
+            if (componentIds.Count == 0 || componentIds == null)
+                throw new NullReferenceException(Properties.Resources.NoChildElements);
+            List<Element> componentElements = componentIds.Select(id => Document.GetElement(id).ToDSType(true))
+                                                           .ToList();
+            return componentElements;
+        }
+
+        private static List<Element> GetChildElementsFromRailings(Autodesk.Revit.DB.Element element)
+        {
+            var railingElement = element as Autodesk.Revit.DB.Architecture.Railing;
+            List<ElementId> railingComponentIds = new List<ElementId>();
+            railingComponentIds.AddRange(railingElement.GetHandRails().ToList());
+            railingComponentIds.Add(railingElement.TopRail);
```

Comment:
Same here, comments for these two lines please incase these need to be updated in the future